From 041774829836e7381cc1123b76a9a595561e2372 Mon Sep 17 00:00:00 2001
From: Stefan Agner <stefan.agner@toradex.com>
Date: Thu, 19 Sep 2019 17:09:58 +0200
Subject: [PATCH 3/4] colibri-imx6ull: use distro boot by default

Use distro boot for raw NAND modules by default. This is required
for TorizonCore.

Signed-off-by: Stefan Agner <stefan.agner@toradex.com>
---
 include/configs/colibri-imx6ull.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/configs/colibri-imx6ull.h b/include/configs/colibri-imx6ull.h
index e2c31cd4e0..9c628ad39e 100644
--- a/include/configs/colibri-imx6ull.h
+++ b/include/configs/colibri-imx6ull.h
@@ -83,10 +83,11 @@
 		"ubi read ${fdt_addr_r} dtb && " \
 		"run fdt_fixup && bootz ${kernel_addr_r} - ${fdt_addr_r}\0" \
 
-#define CONFIG_BOOTCOMMAND "run ubiboot; " \
+#define CONFIG_BOOTCOMMAND \
 	"setenv fdtfile " FDT_FILE " && run distro_bootcmd;"
 
 #define BOOT_TARGET_DEVICES(func) \
+	func(UBIFS, ubifs, 0) \
 	func(MMC, mmc, 0) \
 	func(USB, usb, 0) \
 	func(DHCP, dhcp, na)
-- 
2.27.0

